scrapd/core/apd.py

```diff
@@ -415,7 +455,7 @@ def parse_deceased_field_common(split_deceased_field, fleg):
 
     # Extract and clean up DOB.
     raw_dob = split_deceased_field[-1].strip()
-    d[Fields.DOB] = date_utils.clean_date_string(raw_dob, True)
+    d[Fields.DOB] = clean_date_string(raw_dob, True)
```

Comment:
Is this still useful? As you use Python objects for manipulating the dates, I believe you could use the `parse_date()` function directly. 

You may even be able to get rid of the `clean_date_string()` function now.

Reply:
I made the change, but it did result in separate DOB parsing commands for the "page" and "twitter" processes.